recipes-kernel/linux/files/prplmesh.cfg

```diff
@@ -0,0 +1,28 @@
+CONFIG_TUN=y
+CONFIG_DUMMY=y
+CONFIG_BRIDGE=y
+CONFIG_BRIDGE_NETFILTER=y
+CONFIG_BRIDGE_NF_EBTABLES=m
```

Comment:
Maybe better link it in instead of module?

Reply:
Kernel menuconfig does not suggest such variant but `m` only